packages/test-utils/src/wrapper.js

```diff
@@ -531,6 +531,116 @@ export default class Wrapper implements BaseWrapper {
     this.vnode = this.vm._vnode
   }
 
+  /**
+   * Sets element value and triggers input event
+   */
+  setValue (value: any) {
+    const el = this.element
+
+    if (!el) {
+      throwError('cannot call wrapper.setValue() on a wrapper without an element')
+    }
+
+    const tag = el.tagName
+    const type = this.attributes().type
+    const event = 'input'
+
+    if (tag === 'SELECT') {
+      throwError('wrapper.setValue() cannot be called on select')
+    } else if (tag === 'INPUT' && type === 'checkbox') {
+      throwError('wrapper.setValue() cannot be called on checkbox. Use wrapper.setChecked() instead')
+    } else if (tag === 'INPUT' && type === 'radio') {
+      throwError('wrapper.setValue() cannot be called on radio. Use wrapper.setChecked() instead')
+    } else if (tag === 'INPUT' || tag === 'textarea') {
+      // $FlowIgnore
+      el.value = value
+      this.trigger(event)
+    } else {
+      throwError('wrapper.setValue() cannot be called on this element')
+    }
+  }
+
+  /**
+   * Checks radio button or checkbox element
+   */
+  setChecked (checked: boolean) {
+    if (typeof checked !== 'undefined') {
```

Comment:
I think we should default to true, so you can call setChecked() and it will set the element to checked. What do you think?

Reply:
It is set to true if no argument is passed. It isn't set on the parameters as a default but it behaves the way you suggest, there is a test for this specific case.